platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/linking/OrphanBufferingLinker.java

```diff
@@ -90,19 +76,17 @@ private static void parentNoLongerMissing(final ChildEvent child, final Hash par
         }
     }
 
-    private void orphanPurged(final EventDescriptor key, final ChildEvent orphan) {
+    private static void orphanPurged(final EventDescriptor key, final ChildEvent orphan) {
         // this should never happen. an events parents should become ancient and at that point it will no longer be an
         // orphan
         if (orphan == null) {
             logger.error(LogMarker.EXCEPTION.getMarker(), "Null orphan, descriptor: {}", key);
             return;
         }
         orphan.orphanForever();
-        logger.error(LogMarker.EXCEPTION.getMarker(), "Purging an orphan: {}", orphan.getChild());
+        orphan.getChild().getBaseEvent().exitIntakePipeline();
 
-        // notify the intake monitor that this orphan event was purged, and thus is done traversing the intake pipeline
-        intakePipelineManager.eventThroughIntakePipeline(
-                orphan.getChild().getBaseEvent().getSenderNodeId());
+        logger.error(LogMarker.EXCEPTION.getMarker(), "Purging an orphan: {}", orphan.getChild());
```

Comment:
I'm not sure this should log an exception. An attacker can trivially create events that will provoke this log. Also possible without an attacker if nodes are starting to fall behind.